services-js/commissions-app/src/server/email/EmailContent.ts

```diff
@@ -0,0 +1,66 @@
+import fs from 'fs';
+import Handlebars from 'handlebars';
+import mjml2html from 'mjml';
+
+require('./handlebars-helpers');
+
+export interface TemplateData {
+  commissions: Array<string>;
+  email: string;
+  newId: string;
+}
+
+interface Templates {
+  MJML: any;
+  TEXT: any;
+}
+
+interface RenderedEmailBodies {
+  html: string;
+  text: string;
+}
+
+const templates = {
+  applicant: readTemplateFiles('applicant'),
+  policyOffice: readTemplateFiles('policyOffice'),
+};
+
+export default class EmailContent {
+  handlebarsApplicantTemplates: any;
+  handlebarsPolicyOfficeTemplates: any;
+
+  constructor() {
+    this.handlebarsApplicantTemplates = readTemplateFiles('applicant');
```

Comment:
This seems redundant with the `templates` global above.

Between the two, I prefer having all the work happen inside the class when an object is constructed or initialized, rather than at module load time.

Reply:
That **is** totally redundant - thanks for catching!